db/seeds.rb

```diff
@@ -1,3 +1,102 @@
-framework = Framework.create!(short_name: 'cboard8', name: 'Cheese Board 8')
-framework.lots.create!(number: '1', description: 'Soft cheese')
-framework.lots.create!(number: '2', description: 'Hard cheese')
+supplier = Supplier.find_or_create_by!(name: 'Bobs Cheese Shop')
+due_date = Date.today.next_month.change(day: 7)
+
+
+framework_unstarted = Framework.find_or_create_by!(short_name: 'cboard8', name: 'Cheese Board 8')
+Agreement.find_or_create_by!(supplier: supplier, framework: framework_unstarted)
+Task.find_or_create_by!(
```

Comment:
Is it worth explicitly setting the status to `:unstarted` here?

Reply:
I think I probably didn't do this because `:unstarted` is the initial state. That said, I've just spotted there's a typo in the state machine! –

https://github.com/dxw/DataSubmissionServiceAPI/blob/b423497cef3af84fd1e955139e4460b7958def4a/app/models/task.rb#L6

I'll set the state explicitly and also correct the typo in a separate commit.